src/modules/avformat/producer_avformat.c

```diff
@@ -3741,7 +3744,6 @@ static int producer_get_frame(mlt_producer producer, mlt_frame_ptr frame, int in
     // If cache miss
     if (!self) {
         self = calloc(1, sizeof(struct producer_avformat_s));
-        producer->child = self;
```

Comment:
Do you recall why this was removed? This was trying to reconnect these in the same manner as `mlt_producer_init(producer, self)` does in `producer_avformat_init()`. It seems like the correct thing to do even tho a brief analysis tells me this may never be used.

Reply:
It is never used - and it can never be used because the pointer might be deleted by the cache if the cache limit is hit. It essentially  becomes a dangling pointer when the cache destroys it. It was being used by probe because I did not realize the cache could delete it when I wrote the probe function. So I remove this line so that nobody else will try to use it.